src/components/contextual/pages/pool/PoolChart.vue

```diff
@@ -33,7 +33,7 @@ type Props = {
   // eslint-disable-next-line vue/require-default-prop -- TODO: Define default prop
   tokensList?: string[];
   // eslint-disable-next-line vue/require-default-prop -- TODO: Define default prop
-  poolType?: PoolType;
+  poolType?: string;
```

Comment:
We could continue using PoolType and import from SDK here.

Reply:
The PoolType in the SDK is just a string type. The PoolType we had was an enum, they're not the same.